tests/test.validator.js

```diff
@@ -610,3 +610,41 @@ describe('Validator.detectTypeFromLayout()', function() {
       });
   });
 });
+
+
+describe('Validator.run()', function() {
+
+  it('should run extractMetaData()', () => {
+    var addonValidator = new Validator({_: ['foo'], metadata: true});
+    var fakeMetaData = {type: 1, somethingelse: 'whatever'};
+    addonValidator.toJSON = sinon.stub();
```

Comment:
A few newlines in here would help stuff breathe a bit, maybe after the `var` declarations and before the `return`.